src/main/resources/AntwoordenModel_v2_2_2.csv

```diff
@@ -188,10 +188,10 @@ Ja,Ja,Nee,Nee,,Nee,Een_ouder,,,,Ja,,,,,Ja_ouder2,,43o2,V,Nee,Nee,Nee,Ja,4370,Ing
 Ja,Ja,Nee,Ja,Ja,Nee,Een_ouder,,,,Ja,,,,,Ja_ouder2,,43ao2,V,Nee,Nee,Nee,Ja,4371,Ingezeten - minderjarig - adoptie NL akte - heeft één ouder - gehuwd/partnerschap - ouder onbevoegd - soort gezag is V - de partner van ouder2 is voogd
 Ja,Ja,Nee,Nee,,Nee,Een_ouder,,,,Ja,,,,,Ja_beiden,,44,G,Nee,Nee,Nee,Nee,4450,Ingezeten - minderjarig - heeft één ouder - ouder gehuwd/partnerschap - ouder en partner onbevoegd - soort gezag is G - tijdelijk geen gezag
 Ja,Ja,Nee,Ja,Ja,Nee,Een_ouder,,,,Ja,,,,,Ja_beiden,,44a,G,Nee,Nee,Nee,Nee,4451,Ingezeten - minderjarig - adoptie NL akte - heeft één ouder - ouder gehuwd/partnerschap - ouder en partner onbevoegd - soort gezag is G - tijdelijk geen gezag
-Ja,Ja,Nee,Nee,,Nee,Twee_ouders,,,,,,,,,,,45e,N,Nee,Nee,Nee,Nee,4500,gezag is niet te bepalen omdat bij bepaling huwelijk/partnerschap relevante gegevens ontbreken. Het gaat om de volgende gegevens: xxxx
-Ja,Ja,Nee,Ja,Ja,Nee,Twee_ouders,,,,,,,,,,,45ae,N,Nee,Nee,Nee,Nee,4501,gezag is niet te bepalen omdat bij bepaling huwelijk/partnerschap relevante gegevens ontbreken. Het gaat om de volgende gegevens: xxxx
-Ja,Ja,Nee,Nee,,Nee,Twee_ouders,Nee,,,,,,,,,,45ee,N,Nee,Nee,Nee,Nee,4510,gezag is niet te bepalen omdat bij de bepaling van bevoegdheid relevante gegevens ontbreken. Het gaat om de volgende gegevens: xxxx
-Ja,Ja,Nee,Ja,Ja,Nee,Twee_ouders,Nee,,,,,,,,,,45aee,N,Nee,Nee,Nee,Nee,4511,gezag is niet te bepalen omdat bij de bepaling van bevoegdheid relevante gegevens ontbreken. Het gaat om de volgende gegevens: xxxx
+Ja,Ja,Nee,Nee,,Nee,Twee_ouders,,,,,,,,,,,45e,N,Nee,Nee,Nee,Nee,4500,gezag is niet te bepalen omdat bij bepaling huwelijk/partnerschap relevante gegevens ontbreken. Het gaat om de volgende gegevens: %s
```

Comment:
in de toelichting staat nu "bij bepaling huwelijk/partnerschap". Deze situatie kan bijvoorbeeld (vooral?!) voorkomen wanneer een van de ouders niet staat ingeschreven in de BRP (burgerservicenummer ontbreekt). Dan kan de PL van de ouder niet worden opgezocht en dus ook niet gegevens over huwelijk/partnerschap van de ouder. Dan denk ik dat de toelichting wel onduidelijk wordt wanneer het resultaat is: `gezag is niet te bepalen omdat bij bepaling huwelijk/partnerschap relevante gegevens ontbreken. Het gaat om de volgende gegevens: burgerservicenummer van ouder2`. 

Dit is namelijk geen gegeven over huwelijk/partnerschap. En daarom denk ik onduidelijk wanneer je niet weet dat het gaat om huwelijk partnerschap van de ouders.

Misschien aanpassen naar: `gezag is niet te bepalen omdat bij bepaling huwelijk/partnerschap van de ouders relevante gegevens ontbreken. Het gaat om de volgende gegevens: burgerservicenummer van ouder2`. (ik heb "van de ouders" toegevoegd)